app/package.json

```diff
@@ -21,10 +21,12 @@
     "@rocket.chat/sdk": "^1.0.0-alpha.42",
     "axios": "^0.26.1",
     "bootstrap": "^5.1.3",
+    "crypto-js": "^4.1.1",
```

Comment:
`crypto-js` is not required since we will not handle the encrypted authentication in RC4Community. Please remove any related lines of code/ functions which make use of this.